Automattic/jetpack
# WooCommerce Analytics: deprecate feature

#### Changes proposed in this Pull Request:

Originally we created the WooCommerce Analytics feature to bring more information to Store Stats available in the WordPress.com Stats dashboard:
https://wordpress.com/store/stats/orders/day/

We're not currently working on this Store Stats screen and may now be able to use the WooCommerce plugin directly to access that information.

#### Jetpack product discussion

* Internal reference: pj9Io-2ZI-p2

#### Does this pull request change what data or activity we track or use?

* No. Since we're deprecating the feature we'll be able to delete the existing support documents and privacy information.

#### Testing instructions:

* Start from a site running Jetpack Stable and the WooCommerce plugin.
* Go to Jetpack > Modules, and notice that the WooCommerce Analytics feature is active.
* Switch to this branch.
* Go to Jetpack > Modules again; the feature should be gone.
* The feature should not appear when running `wp jetpack module list` either.
* No errors should appear in your logs.

#### Proposed changelog entry for your changes:

* WooCommerce Analytics: deprecate feature
